softicar/platform
# PLAT-956 Added Additional Forbidden References for Eclipse

- note, manual changes are only done to build.gradle
- other files are generated by the Buildship plug-in

### Test Plan:
- create a new class without any imports
- enter each of the following class names and press Ctrl+Space
  - Iterator
  - List
  - Log
  - Predicate
  - ...
- observe only the desired import is suggested
- also observe that existing platform code still compiles in Eclipse
